lib/db/schema/patch-047-048.sql

```diff
@@ -0,0 +1,80 @@
+CREATE TABLE IF NOT EXISTS `signinCodes` (
+  hash BINARY(32) NOT NULL PRIMARY KEY,
+  uid BINARY(16) NOT NULL,
+  createdAt BIGINT UNSIGNED NOT NULL,
+  INDEX signinCodes_createdAt (createdAt)
+) ENGINE=InnoDB;
+
+CREATE PROCEDURE `createSigninCode_1` (
+  IN `hashArg` BINARY(32),
+  IN `uidArg` BINARY(16),
+  IN `createdAtArg` BIGINT UNSIGNED
+)
+BEGIN
+  INSERT INTO signinCodes(hash, uid, createdAt)
+  VALUES(hashArg, uidArg, createdAtArg);
+END;
+
+CREATE PROCEDURE `useSigninCode_1` (
+  IN `hashArg` BINARY(32)
+)
+BEGIN
+  DECLARE EXIT HANDLER FOR SQLEXCEPTION
+  BEGIN
+    ROLLBACK;
+    RESIGNAL;
+  END;
+
+  START TRANSACTION;
+
+  SELECT email
+  FROM accounts
+  WHERE uid = (
+    SELECT uid
+    FROM signinCodes
+    WHERE hash = hashArg
+  );
+
+  DELETE FROM signinCodes
+  WHERE hash = hashArg;
+
+  COMMIT;
+END;
+
+CREATE PROCEDURE `expireSigninCodes_1` (
+  IN `olderThan` BIGINT UNSIGNED
+)
+BEGIN
+  DELETE FROM signinCodes
+  WHERE createdAt < olderThan;
+END;
+
+CREATE PROCEDURE `deleteAccount_13` (
```

Comment:
Should update the call to use new procedure, https://github.com/mozilla/fxa-auth-db-mysql/blob/master/lib/db/mysql.js#L519